### PR TITLE
llvm: patch missing cstdint include

### PR DESCRIPTION
**Pull request by haampie**
Seems to apply almost universally

### CODE REVIEW

**Comment by skosukhin** [comment]
> llvm: patch missing cstdint.h include

I am pretty sure there's no such standard include :)

**Comment by haampie** [comment]
It's a header used in (C++)++

File: var/spack/repos/builtin/packages/llvm/package.py [hunk 1]
```diff
@@ -428,6 +428,12 @@ class Llvm(CMakePackage, CudaPackage):
         when="@14:15",
     )
 
+    # missing <cstdint.h> include
```
Comment (haampie) [inline var/spack/repos/builtin/packages/llvm/package.py]:
```suggestion
    # missing <cstdint> include
```

**Review by tldahlgren — APPROVED** [review]
LGTM

**Review by skosukhin — APPROVED** [review]

**Pull request merged**